.env.example

```diff
@@ -7,6 +7,9 @@ WP_ENV=development
 WP_HOME=http://example.com
 WP_SITEURL=${WP_HOME}/wp
 
+# Optional variables
```

Comment:
Interesting idea, although I feel like almost every case it will need a port number added, so I'm not sure the implication that it might be ok to leave it will help ppl. Although that may just be my own experience